packages/flutter/test/semantics/semantics_test.dart

```diff
@@ -241,18 +242,42 @@ void main() {
 
 class TestRender extends RenderProxyBox {
 
-  TestRender({ this.action, this.isSemanticBoundary, RenderObject child }) : super(child);
+  TestRender({
+    this.tapAction: false,
+    this.longPressAction: false,
+    this.scrollLeftAction: false,
+    this.scrollRightAction: false,
+    this.scrollUpAction: false,
+    this.scrollDownAction: false,
+    this.isSemanticBoundary,
+    RenderObject child
+  }) : super(child);
+
+  bool tapAction;
+  bool longPressAction;
+  bool scrollLeftAction;
+  bool scrollRightAction;
+  bool scrollUpAction;
+  bool scrollDownAction;
```

Comment:
The names of these fields would read better with "has" or "with" as a prefix.

Reply:
Done.